### PR TITLE
Fixed forestry incompatibility

### PR DESCRIPTION
**Pull request by Tencao**
Fixed forestry incompat, bumped minecraft and forge version requirement to match forestry as well as update mappings.

### CODE REVIEW

**Comment by ghost** [comment]
hey ah, when this version will be released on curseforge? thanks
btw @Tencao , can you share the fixed jar with me please, i will only play it. thank you

**Comment by axieum** [comment]
Can confirm this build runs perfectly fine for MC 1.12.2 running latest Forestry.

**Review by kdb424 — APPROVED** [review]
Tested changes on my end. Clean edit. Looks like a safe merge.

**Comment by ghost** [comment]
very good! we will be waiting for it on CF.

**Comment by Daihok41** [comment]
Its been almost a week anyone know when this fix will hit curseforge?

**Comment by kenijey** [comment]
you can use this jar if you will only play it.
[morebees-1.12.2-1.4.3.zip](https://github.com/Lach01298/MoreBees/files/1463484/morebees-1.12.2-1.4.3.zip)

yeah its a zip, rename it to a jar then put in mods folder.

**Comment by Daihok41** [comment]
awesome thank you.

**Comment by Tencao** [comment]
I would have linked the jar as well but since I don't know how the mod author treats sharing custom versions, I decided to hold off on that.

**Comment by axieum** [comment]
@Tencao I had the same dilemma. I'm still surprised it's not live on Curse yet. Also, I'll say thank you on everyone's' behalf for this pull request!

**Comment by SethWilliamson** [comment]
Careful what you release @kenijey 
The mod is licensed LGPLv3, therefore you cannot release your fork / patched version under more restrictive terms like "if it is for only you to play." Your file is freely redistributable.

@Tencao -- The license on the mod's project page spells out the author's permissions regarding custom versions. It is [LGPLv3](https://www.gnu.org/licenses/lgpl-3.0.en.html) which incorporates [GPLv3](https://www.gnu.org/licenses/gpl-3.0.html) with some modifications. Sections 4, 5, and 6 of the GPL3 are probably of most interest to you.

In short, anyone can fork it and make changes, but their version and changes must carry the same license. If you want, you could even freely post a new version on Curse.

His licence can be found on the right under "About this project"
https://minecraft.curseforge.com/projects/more-bees

**Comment by ghost** [comment]
i just helped to ppl that want to play this mod. i dont care anything else. **i deleted the message that contains mod file.** as i said, i dont care.

**Comment by SethWilliamson** [comment]
@kenijey -- Hope I didn't sound like I was chastising you for posting or something. Not my intention. You're more than free to do so and I'm one of the folks that is thankful that you did. That's one of the things that makes open source projects so great is that others can contribute and everyone benefits. I just wanted to make sure you were aware that people are free to redistribute your patched version under the upstream license, and if you did have a problem with that, you could remove it. If you truly don't care what people do with your file and just want to help, I'm sure people would be really happy if you re-posted your file. I know I would. :) Thanks for posting it in the first place!

**Comment by Tencao** [comment]
I understand the licence, but I would rather get the authors approval of the changes I've made first if at all possible.

**Comment by Darianna77** [comment]
Mhh can anyone share the fixed file with me? Have the Mod in my Modpack wich have get an update and now its broken because of the crash...cant disable the mod because i use it very much :(
It would be glad when i can get the file so that i can play again...

**Comment by kdb424** [comment]
If you want the file then compile it yourself. As stated by request of the mod author, you can not distribute custom versions. If you can not comply with that licence, you break literal laws and can be held accountable, and more to the point, you disrespect the mod author. Please do not ask for others to break the laws, and please do not break them yourself. I am not the mod author, but simply a software developer that would like licences to be followed. Good day.

**Comment by SethWilliamson** [comment]
@kdb424 Where are you seeing the author's request not to distribute custom versions? Under his license, he has specifically given permission in advance to do just that. (See sections on conveying modified versions.)

**Comment by kdb424** [comment]
Sorry. I was in a similar discussion elsewhere ironically enough. There was however a licencing issue possibly above and must have mixed things up. lGPL does in fact allow for redistribution with modifications. It would however be more respectful to link front somewhere else, no laws would be broken. Sorry for the mistake.

**Comment by Lach01298** [comment]
As i have stated before I've given up on this mod but i'm am willing to hand over ownership to someone who will maintain it.
also another reason i haven't released or tested this branch is because my old computer died and i have to setup a new dev environment but i haven't bothered to do so yet.

**Comment by Lach01298** [comment]
@Tencao do you want to take over the mod?

**Comment by Tencao** [comment]
I don't mind, can't promise to add anything new but I'll at least do my best to fix any issues I find

**Comment by Lach01298** [comment]
Ok just give your curse forge username and i'll give it to you

**Comment by Tencao** [comment]
Curse forge username should be Tencao
